libcloud/common/ovh.py

```diff
@@ -115,17 +116,23 @@ def request_consumer_key(self, user_id):
 
     def get_timestamp(self):
         if not self._timedelta:
-            url = 'https://%s/%s/auth/time' % (API_HOST, API_ROOT)
+            url = 'https://%s%s/auth/time' % (API_HOST, API_ROOT)
```

Comment:
can you just clarify for me that it was intentional to drop `/` in between %s and %s